### ISSUE TITLE
Using Fastlane for CI/CD

### ISSUE THREAD

**Issue by siepra** [TryQuiet/quiet#1833]

**Comment by siepra** [comment]
@holmesworcester @leblowl I'd like to prioritize it, especially that we'll probably be doing a lot of testing deployments soon while working on iOS

**Comment by holmesworcester** [comment]
We'd run it on our own infrastructure? Any worries or downsides?

**Comment by siepra** [comment]
There's no external infrastructure needed. We'd set up our "lanes" and then just call them from within GitHub actions. We'll also be able to run them locally, which right now is difficult as it requires manually firing specific commands in order

**Comment by siepra** [comment]
Also there's no risk as we'll always have our existing workflows setup as backup

**Comment by holmesworcester** [comment]
Sounds good to me!

**Comment by leblowl** [comment]
I agree, seems like a good idea, especially if we can integrate it with Github actions.

**Comment by siepra** [comment]
I'll move it to the top of current sprint then

**Comment by siepra** [comment]
Due to Fastlane not working smooth with our current CI setup, I decided to postpone completion of this task until more pressing tasks are being done.

**Comment by holmesworcester** [comment]
@siepra how urgent is this? what do we lose currently by not having this?
